tasks/run.js

```diff
@@ -78,6 +78,7 @@ function runLocal(opts) {
 		}
 
 		if(opts.nodemon) {
+			args.push('--ignore', 'public/');
```

Comment:
I don't think we need to manually listen to `head.*` changes anymore, as n-express was updated to monitor changes. Though I've no idea where that is now...